.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
+dist: bionic
 python:
 - '3.6'
 - '3.7'
 - '3.8'
+- '3.9'
+- '3.10.0'
+- '3.10.1'
```

Comment:
Why explicitly test both 3.10.0 and 3.10.1?

Reply:
Because there is no Travis option for just `3.10`; only `3.10.0`, `3.10.1` and `3.10-dev`.  I'm okay with dropping `3.10.0` but had it in there because conda doesn't have a 3.10.1 build yet so I wanted to be able to make sure that anything that went wrong was not because of a 3.10.0 v. 3.10.1 difference.

https://docs.travis-ci.com/user/languages/python/

Reply:
To clarify, nothing went wrong with 3.10.1, correct?